examples/subscription/create_auto_plan.rb

```diff
@@ -0,0 +1,48 @@
+require_relative '../boot'
+
+# Create a Subscription Plan (Auto charge)
+#
+# You need to set your AccountCredentials (EMAIL, TOKEN) in the application config
+#
+# P.S: See the boot file example for more details.
+
+plan = PagSeguro::SubscriptionPlan.new(
+  charge: 'auto', # Automatic Subscription Plan must always use auto charge type.
+
+  name: 'Seguro contra roubo do Notebook',
+  details: 'Taxa referente ao seguro contra roubo de Notebook',
+  amount: 100.0,
+  max_amount: 2400.0,
+  period: 'Monthly',
+  final_date: Time.new(2017, 2, 28, 20, 20),
+
+  redirect_url: 'http://example.com/redirect',
+  review_url: 'http://example.com/review',
+  reference: 'AUTOPLAN123',
+  sender: {
+    name: 'Nome do Cliente',
+    email: 'client@example.com',
+    phone: { area_code: 11, number: 123456 },
+  },
+  sender_address: {
```

Comment:
Do you want to change this to keep the compatibility with PR #200 or do you want release this way and change after?